suitability/toolboxes/scripts/NAMDownload.py

```diff
@@ -1,228 +1,170 @@
-        arcpy.ImportToolbox(mstToolboxPath, "mds")
-
-        #Get present date
+        now_time = time(int(datetime.utcnow().strftime("%H")), int(datetime.utcnow().strftime("%M")), int(datetime.utcnow().strftime("%S")))
+        print("UTC time is (now_time): %s" % now_time)
+        
         patternDate = '%Y%m%d'
-        global stringDateNow
-        if stringDateNow == None:
-            stringDateNow = datetime.utcnow().strftime(patternDate)
-        print("Target date: " + str(stringDateNow))
-
-        noaaURLBase = r"http://nomads.ncep.noaa.gov/dods/nam/nam%s/nam" + paramDL
-        noaaURL = noaaURLBase % stringDateNow
-        #filename = "nam%s1hr00z.nc" % stringDateNow
-
-        print("Building forecast data...")
-        netcdfDataFileBase = os.path.join(netcdfFolder, r"nam%s" + paramFN + ".nc") % stringDateNow
-        if checkOSExists(netcdfDataFileBase):
-            print("deleting existing " + str(netcdfDataFileBase))
-            removeFile(netcdfDataFileBase)
-        forecastVariable = r"rh2m;tcdcclm;tmpsfc;hgtclb;vissfc;ugrd10m;vgrd10m;ugrdmwl;vgrdmwl;snodsfc;gustsfc;apcpsfc"
-        operationalData = os.path.join(maowOperationalWeatherGeodatabase, "OperationalData")
-        result = process(noaaURL, forecastVariable, netcdfDataFileBase, operationalData)
-
-        print("Building wind data...")
-        netcdfWindFileBase = os.path.join(netcdfFolder, r"nam%s" + paramFN + "Wind.nc") % stringDateNow
-        if checkOSExists(netcdfWindFileBase):
-            print("deleting existing " + str(netcdfWindFileBase))
-            removeFile(netcdfWindFileBase)
-        windVariable = r"ugrd10m;vgrd10m"
-        operationalWind = os.path.join(maowOperationalWeatherGeodatabase, "OperationalWind")
-        result = process(noaaURL, windVariable, netcdfWindFileBase, operationalWind)
-
-        print("Done.")
-
-    except NAMDownloadException as e:
-        print("Error in main: \n" + str(e.tb))
-
-# get the present time in utc.
-# set times are set around the times that the data is released by NOAA
-now_time = time(int(datetime.utcnow().strftime("%H")),
-                int(datetime.utcnow().strftime("%M")),
-                int(datetime.utcnow().strftime("%S")))
-print("Now time: " + str(now_time))
-if now_time >= time(2, 50, 00) and now_time < time(8, 50, 00):
-    download("1hr00z", "1hr_00z")
-
-elif now_time >= time(8, 50, 00) and now_time < time(14, 50, 00):
-    download("1hr06z", "1hr_06z")
-
-elif now_time >= time(14, 50, 00) and now_time < time(21, 00, 00):
-    download("1hr12z", "1hr_12z")
-
-elif ((now_time >= time(21, 00, 00) and now_time <= time(23, 59, 59)) or (now_time >= time(00, 00, 00) and now_time <= time(2, 49, 59))):
-    download("1hr18z", "1hr_18z")
+        patternTime = '%H:%M:%S'
+        stringDateNow = datetime.utcnow().strftime(patternDate)
+        stringTimeNow = datetime.utcnow().strftime(patternTime)
+        
+        if now_time >= time(02,50,00) and now_time < time(8,50,00):
```

Comment:
GitHub highlighting is correctly showing this leading zero as a syntax error in Python 3 (in Python 2 it is interpreted as an octal(base 8) - probably not what is intended either)